lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -29,7 +29,9 @@
             required: true
             choices: docker_swarm
         host:
```

Comment:
needs to be renamed to `docker_host` or we'll get this warning:

```
 [WARNING]:  * Failed to parse /home/morph/ansible/docker_swarm.yml with auto plugin: 'Requested entry (plugin_type:
inventory plugin: docker_swarm setting: docker_host ) was not defined in configuration.'
```

Applies to examples too

Reply:
Done. It would also be enough to fix line 151, but I think `docker_host` is somewhat better than `host` since it's the same name for the `docker_*` modules.